support-frontend/assets/pages/paper-subscription-checkout/components/paperCheckoutForm.jsx

```diff
@@ -267,9 +267,11 @@ function PaperCheckoutForm(props: PropTypes) {
                   css={controlTextAreaResizing}
                   id="delivery-instructions"
                   label="Delivery instructions"
+                  supporting="Please let us know any details to help us find your property (door colour, any access issues) and the best place to leave your newspaper. For example, 'Front door - red - on Crinan Street, put through letterbox'"
                   maxlength={250}
                   value={props.deliveryInstructions}
                   onChange={e => props.setDeliveryInstructions(e.target.value)}
+                  optional
```

Comment:
Sorry if I've missed it but I can't see the value of the `optional` prop being set anywhere, it looks like this should be a `boolean` from the Design System [documentation](https://theguardian.design/2a1e5182b/p/663879--select-box/b/59d4fd) so it should be explicitly set to `true`.

Reply:
You're so right, but the linter yells at me if I try to do that!
<img width="642" alt="Screen Shot 2021-03-02 at 14 21 53" src="https://user-images.githubusercontent.com/16781258/109662208-b6463d00-7b62-11eb-8a57-5927bdb4b900.png">

Reply:
I think you only explicitly set it for false in this paradigm, and not setting it is like setting it to false.

Reply:
OK I see, if if it's present it's `truthy` and if it's absent is `falsey`